app/views/mailing_list/steps/_name.html.erb

```diff
@@ -1,9 +1,16 @@
-<h1>Get personalised information and updates about getting into teaching</h1>
+<h1>Get personalised guidance to your inbox</h1>
 
 <p>
-  We know you’ll only want to receive updates that help you 
-  take your next steps into teaching. We'll use the information 
-  you provided to make sure emails we send are relevant to you.
+  Get one step closer to the classroom with guidance tailored to you. Find out how to:
+</p>
+
+<ul>
+  <li> become a teacher
+  <li> get one-to-one support
```

Comment:
```suggestion
  <li> become a teacher</li>
  <li> get one-to-one support</li>
```

While not _absolutely necessary_ it's usually a good idea to close `<li>` tags

Reply:
ah thanks @peteryates